site/kube-addon.md

```diff
@@ -205,21 +207,16 @@ UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 
 ### <a name="configuration-options"></a> Changing Configuration Options
 
-The default configuration settings can be changed by saving and editing the
-addon YAML before running `kubectl apply`. Additional arguments may be
-supplied to the Weave router process by adding them to the `command:`
-array in the YAML file.
+#### Using `cloud.weave.works`
 
-Some parameters are changed by environment variables; these can be
-inserted into the YAML file like this:
+If you are getting your YAML file from `cloud.weave.works` as described above, then you can customise it by passing some of Weave Net's options, arguments and environment variables as query parameters:
 
-```
-      containers:
-        - name: weave
-          env:
-            - name: IPALLOC_RANGE
-              value: 10.0.0.0/16
-```
+  - `version`: Weave Net's version. Default: `latest`, i.e. latest release. *WARNING*: Backward compatibility of the YAML is not guaranteed. If you lock Weave Net's version using something else than `latest` for this query parameter, please make sure you do so with the latest release, save a copy of the YAML and use this copy instead of downloading it every time from `cloud.weave.works`.
+  - `known-peers`: comma-separated list of hosts. Default: empty.
```

Comment:
My first attempt was aiming for conciseness, but probably failed to be clear.
Also, if we left it undocumented, and users store a YAML with `latest` in it, this could also cause troubles for them over time.
Would you rather have me detail it more as per my previous comment?
WDYT @abuehrle, @bboreham?